src/components/McuUpdateDialogView.tsx

```diff
@@ -231,7 +257,14 @@ Are you sure you want to continue?`,
                     <DialogButton
                         variant="primary"
                         onClick={onWriteStart}
-                        disabled={writing || writingSucceed || writingFail}
+                        disabled={
+                            writing ||
+                            writingSucceed ||
+                            writingFail ||
+                            (programmingOptions.length > 1 &&
+                                !chosenTarget &&
+                                !!mcubootFwPath)
```

Comment:
Looks a bit complex, how this one should be tested? looks like it does overlap with existing mcuboot programming?